Source/WebCore/platform/graphics/gstreamer/eme/CDMThunder.cpp

```diff
@@ -501,7 +501,12 @@ void CDMInstanceSessionThunder::requestLicense(LicenseType licenseType, KeyGroup
 
     // FIXME: UUID or system ID?
     auto instance = cdmInstanceThunder();
-    ASSERT(instance);
+    if (!instance) {
+        GST_WARNING("no instance, bailing out");
+        ASSERT_NOT_REACHED_WITH_MESSAGE("no CDMInstanceThunder object");
```

Comment:
Should we really assert here? a GC'd CDMInstance doesn't seem like a programmer (at least on WebKit  side :)) error.

Reply:
I was wondering the same thing myself, but in the end I had decided to keep the "same behavior". No strong opinion so I removed it.